pkgs/racket-doc/scribblings/reference/subprocess.scrbl

```diff
@@ -111,6 +111,19 @@ The @racket[subprocess] procedure returns four values:
 explicitly closed, usually with @racket[close-input-port] or
 @racket[close-output-port].
 
+Example:
+
+@racketblock[
+(define-values (k out in err)
+  (subprocess #f #f #f "/bin/ls" "-l"))
+(displayln (format "stdout:\n~a" (port->string out)))
```

Comment:
FWIW, there's `printf` which you can used instead of `displayln` + `format`.

Reply:
Ahhh, I used to see the `printf` as a c style code...